app/presenters/claim_csv_presenter.rb

```diff
@@ -22,7 +22,7 @@ def sorted_and_filtered_state_transitions
   def parsed_journeys
     journeys.map do |journey|
       @journey = clean_deallocations(journey)
-      Settings.claim_csv_headers.map { |method_call| send(method_call) }
+      Settings.claim_csv_headers.map { |method_call| send(method_call) } if @journey.any?
```

Comment:
**[not a blocker]** Re-setting this instance variable per iteration of `journeys` array suggests (to me) that we should probably have some sort of `ClaimCSV::Journey` presenter that handles all _THE_ journey presentation, leaving this one to be something like:

```ruby
def parsed_journeys
  journeys.map do |journey|
    ClaimCSV::Journey.new(clean_deallocations(journey)).to_csv
  end
end
```

This is probably more an improvement in the design of the presenter to isolate the claim presentation from the journey presentation.

Reply:
Good suggestion, I've added an issue as we need to get this bugfix out so they can generate their stats